mods/tuxemon/db/technique/status_poison.json

```diff
@@ -10,7 +10,8 @@
   "category": "poison",
   "sort": "meta",
   "effects": [
-    "status"
+    "poison",
+    "status status_poison"
```

Comment:
`"status status_poison"`
This shouldn't be here, it should be on the techniques "suck_poison" and "venemous_tentacle".

Reply:
Check all the techniques that should apply a status.

Reply:
Yeah, you know, that's what I wanted to say. Thanks for repeating it.

Reply:
Also, you should remove "poison" from those techniques.